.circleci/config.yml

```
@@ -0,0 +1,33 @@
+version: 2.1
+
+jobs:
+  test:
+    docker:
+      - image: circleci/node:8.15
```

How should we keep this current?
Should we replace it with `lts`
```suggestion
      - image: circleci/node:8.15
```
/cc @kodefox/infra

---

I think we should leave it that way for now as the renovate will take care of that when we integrate it with this repo too.

---

Well, if we use `lts`, renovate doesn't have to upgrade the image for node, and we have less PR to review, so I think it's not a bad idea.

---

Are you sure about that? If so that would be a great idea, actually.

---

See here, https://docs.renovatebot.com/node/

It doesn't list circle CI yet, but travis is there, so I'm pretty sure we should be able to configure it.

---

I think it should work with CircleCI too.

---

Should we use the latest LTS release?
```json
"node": {
  "supportPolicy": ["lts_latest"]
}
```

---

Or just change the circle config to use lts 🤔

---

I'm good with either the simple version (update the config to always use lts) or also fine with the renovate approach. Will leave this to your judgement!